jetty-ee9/jetty-ee9-servlet/src/test/java/org/eclipse/jetty/ee9/servlet/RegexServletTest.java

```diff
@@ -93,10 +93,10 @@ public void testForward() throws Exception
         String response = _connector.getResponse("GET /ctx/forward/ignore HTTP/1.0\r\n\r\n");
         assertThat(response, containsString(" 200 OK"));
         assertThat(response, containsString("contextPath='/ctx'"));
-        assertThat(response, containsString("servletPath='/Test'"));
-        assertThat(response, containsString("pathInfo='/info'"));
+        assertThat(response, containsString("servletPath='/Test/info'"));
```

Comment:
This seems wrong to me.
If the servlet is mapped to `^/[Tt]est(/.*)?`, then should not the be `servletPath=/Test` and `pathInfo=/info`?